deployment/templates/mongodb-deploymentconfig.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    app: {{ .Values.name }}
+  name: "{{ .Values.name }}-mongodb"
+spec:
+  replicas: 1
+  selector:
+    app: "{{ .Values.name }}-mongodb"
+    deploymentconfig: "{{ .Values.name }}-mongodb"
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: "{{ .Values.name }}-mongodb"
+        deploymentconfig: "{{ .Values.name }}-mongodb"
+    spec:
+      containers:
+      - capabilities: {}
+        env:
+        - name: MONGODB_USER
+          valueFrom:
+            secretKeyRef:
+              key: MONGODB_USER
+              name: "{{ .Values.name }}-configuration"
+        - name: MONGODB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              key: MONGODB_PASSWORD
+              name: "{{ .Values.name }}-configuration"
+        - name: MONGODB_ADMIN_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              key: MONGODB_ADMIN_PASSWORD
+              name: "{{ .Values.name }}-configuration"
+        - name: MONGODB_DATABASE
+          valueFrom:
+            secretKeyRef:
+              key: MONGODB_DATABASE
+              name: "{{ .Values.name }}-configuration"
+        image: ' '
```

Comment:
Does this get immediately overwritten by the image change trigger and that's why it's empty?

Reply:
Correct, will look something like this once deployed
```
image: docker-registry.default.svc:5000/openshift/mongodb@sha256:1dc24b81ceeaefa815f13c7f247b665d9558f42af11a6506f5ccdaa3cf75427d
```